stacks-signer/src/signerdb.rs

```diff
@@ -190,6 +193,7 @@ impl BlockInfo {
         } else {
             self.signed_self.get_or_insert(get_epoch_time_secs());
         }
+        self.processed_time = Some(get_epoch_time_secs());
```

Comment:
This needs to be updated to use the block validate response process time (which will most likely be a nmb secs rather than a timestamp)

Reply:
See #5474